apps/labs/posts/joining_labs.md

```diff
@@ -0,0 +1,71 @@
+
+When I started working on NumPy and SciPy the world was simple: projects with Subversion repositories, mailing lists, and volunteers, and a couple of yearly conferences. Today the landscape looks a lot more complex. This rough sketch attempts to capture a part (all doesn't fit on a 2-D canvas) of how I think it looks from a community or project perspective:
+
+![Image of a word cloud emphasizing the interrelationship of Projects with Assets, Funding and Community.](/posts/joining_labs/community_landscape.png)
+
+Acting effectively within that landscape as a project has become important maintenance work in itself, and working on governance structures or legal agreements is less fun (and thus harder to find volunteers for) than writing code. Quansight Labs will add to this complexity. It also provides an opportunity to help manage that same complexity though, as a home for core developers of many of the projects in the ecosystem. 
+
+### A Quansight Labs perspective
+
+To date, the work at Quansight Labs has mainly focused on innovation and highly experimental prototypes.
+Travis already provided some context on [XND](https://xnd.io/) (a clean, modular implementation of generalized NumPy concepts) and [uarray](https://github.com/Quansight-Labs/uarray) (a Python array interface). XND is already fairly mature while uarray is still very young; however both are at the stage where we can think about adoption pathways. Experimental prototypes with a lot of potential include [python-moa](https://github.com/Quansight-Labs/python-moa) (a Python implementation of Mathematics of Arrays), [metadsl](https://github.com/Quansight-Labs/metadsl) (a framework for creating domain specific language libraries in Python), and [mtypes](https://github.com/Quansight-Labs/mtypes) ("memory types for Python", bridging the Python type system to and from C).
+
+The balance of work at Quansight Labs will gradually shift towards sustaining core projects in the PyData ecosystem.
+Which projects we work on will be driven by both availability of funding and personal interests of people. As well as by input from the community - we want to push forward core projects, so we need to act on priorities that core development teams have. Clearly articulating those priorities isn't always simple. Half a year ago at the NumFOCUS Summit I took stock of the roadmaps for all NumFOCUS projects: 50% of the projects didn't have a roadmap, and of the 50% that did again half said that their roadmaps were not up to date. This is also the kind of thing Quansight Labs can help with. For example, Anthony Scopatz has done a great job in producing roadmap brochures for each of the projects that have been covered in the [Open Source Directions](https://www.quansight.com/open-source-directions) webinars: https://www.quansight.com/projects. In that process he helped improve those project roadmaps tremendously.
```

Comment:
@bskinn These 2 links are broken due to the website redesign. Is there a good replacement here?
CC @rgommers

Reply:
https://www.youtube.com/playlist?list=PLA7hukZmf3AqDMFZD1Oi5cOoe2svop979 for OSD.

https://www.quansight.com/projects can just be removed.

Reply:
@rgommers Are those roadmap brochures archived somewhere we could reference? I know they're out of date, but they might be an interesting historical artifact.

Reply:
Only on GDrive: https://drive.google.com/drive/folders/11prFjz1ZvhBKAM_z8g82lTyNbkjWAE9e. They were floating around as pdf's in Wix I think.

Reply:
I wouldn't bother spending time on that anymore if it requires republishing them I think.

Reply:
We could share the GDrive folder publicly view-only, I guess... That's kind of brittle/smelly/unprofessional, though.

We _can_ serve arbitrary files from Storyblok's assets feature.... We could zip them all into a single archive and link to that?

Reply:
Serving one zipfile of pdf's is fine with me, if that's not much work. Sharing GDrive isn't a good idea indeed.

Reply:
Or, serve the ZIP in the same way we serve blog images, static site data.

Reply:
I assume what needs to be shared are the 22 GSlides in the subfolders you linked to?

Matter of a few minutes, I think, to download each as a PDF, zip, and upload to SB or commit to repo.

Reply:
Not all were published, some are works in progress. There are some that say "This is a community-led project, ...." on the last page, those are mostly fine. Some others say "LEARN MORE sales@quansight.com" - those are no good.

Reply:
Looking at this again, I realize that the roadmaps weren't published as part of this post. So, publishing them now would be a novel effort, and can be delayed for later or skipped entirely. Whatever the eventual choice, it shouldn't block migration of this post. Concur that the projects link can just be removed.